src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.scss

```diff
@@ -11,11 +11,18 @@
 
   .euiSelectableListItem__content {
     cursor: default;
+    padding-left: 0;
+    margin-right: 0;
 
 
   .euiSelectableListItem__text {
-    max-height: 100%; 
+    max-height: 100%;
   }
+  .euiSelectableListItem__icon, .euiSelectableListItem__prepend {
+    margin-right: 0;
+  }
+
+
```

Comment:
nit: i think the formatting of this file is a little off, should we fix the spacing?

Reply:
checking, will remove the extra space in the next push